### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tyler-simons**
Add a bit more detail and clarify for the users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,45 +6,45 @@
 
 # Make your own fork of this app in Codespaces
 
-Step 1:  Click the "Open in Github Codespaces" button above and then click Fork!
+Step 1:  Right click the "Open in Github Codespaces" button above, open the link in a new tab and then click Fork!
```
Comment (tyler-simons) [inline README.md:9]: I did this and then lost my page and had to recreate it :(

File: README.md [hunk 2]
```diff
@@ -6,45 +6,45 @@
...
+_(Eat a 🍪 while you wait)_
 
 Step 2: Press "Create new Codespace"
 
 <img width="521" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/87d24ecc-daf2-4292-a0fa-13e6fc68094e">
 
 _(Check your bluesky while you wait)_
 
-Step 3: After a minute or two, you should see an editor, and then a browser should open to the right (You can resize the site to see the whole app better)
+Step 3: After a minute or two, you should see an editor (it may take a few seconds after the page has loaded), and then a browser should open to the right (You can resize the site to see the whole app better)
```
Comment (tyler-simons) [inline README.md:21]: My page loaded and I got confused when it didn't start `streamlit run`

File: README.md [hunk 3]
```diff
@@ -6,45 +6,45 @@
...
 # Add your own charts
 
-Step 1: Download one of the csvs in this repository
+Step 1: Download one of the csvs above in this repository
 
-Step 2: Upload it to https://vizzu-builder.streamlit.app/
+Step 2: In a new tab, open https://vizzu-builder.streamlit.app/ and upload the CSV you just downloaded.
 
-Step 3: Choose 1 Category and one or more Values columns and hit Update Charts to see the different kinds of charts you can make
+Step 3: Choose 1 Category and one or more Values columns and hit `Create Charts` to see the different kinds of charts you can make.
```
Comment (tyler-simons) [inline README.md:31]: I think you should rename this in the app -- update charts to me means they already exist.

File: README.md [hunk 4]
```diff
@@ -6,45 +6,45 @@
...
 
-Step 3: After a minute or two, you should see an editor, and then a browser should open to the right (You can resize the site to see the whole app better)
+Step 3: After a minute or two, you should see an editor (it may take a few seconds after the page has loaded), and then a browser should open to the right (You can resize the site to see the whole app better)
 
 <img width="1684" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/04f1be56-7b71-4303-81f0-ed7679e4b1a0">
 
 # Add your own charts
 
-Step 1: Download one of the csvs in this repository
+Step 1: Download one of the csvs above in this repository
```
Comment (tyler-simons) [inline README.md:27]: for a second I got confused there wasn't a link for `this`

File: README.md [hunk 5]
```diff
@@ -6,45 +6,45 @@
...
 
 <img width="825" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/868d5ac9-dafd-4fd2-af42-4a05873e81c7">
 
-Step 4: Pick a chart you like and expand the "Show code" expander and press the Copy button
+Step 4: Pick a chart you like and expand the "Show code" expander and press the Copy button at the top right of the section.
 
 <img width="519" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/1b3f24c7-b88a-4d08-a4e2-1a127e26eca4">
 
-Step 5: Go back to the Codespaces instance, and add a new file in the pages/ folder
+Step 5: Go back to the Codespaces instance, and add a new file in the pages/ folder called `charts.py`.
```
Comment (tyler-simons) [inline README.md]: better to give clear direction and make sure they create a .py file.

File: README.md [hunk 6]
```diff
@@ -6,45 +6,45 @@
...
 
 <img width="330" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/47c3d21e-15bd-4e2a-a86c-4fbe70b939f6">
 
-Step 6: Paste the code for your chart into the new file. You should now see a new entry in the sidebar of your app
+Step 6: Paste the code for your chart into the new file and save it. Re-run your app by pressing `r`. You should now see a new entry in the sidebar of your app
```
Comment (tyler-simons) [inline README.md:43]: I think they need to do this, right?

File: README.md (hunk 1 above)
Comment (sfc-gh-zblackwood) [reply README.md:9]: Ooh, good call -- I'll update the link to open in a new tab instead

File: README.md (hunk 3 above)
Comment (sfc-gh-zblackwood) [reply README.md:31]: Yeah, that's a good call. Do you think 'create charts' is confusing when they already exist? If you upload a new dataset, or change the filter, should it always say "Create charts"? Or should I use session state or something to make it first say "Create charts", and then "update charts"?

File: README.md (hunk 6 above)
Comment (sfc-gh-zblackwood) [reply README.md:43]: Yeah, good call

File: README.md (hunk 1 above)
Comment (blackary) [inline README.md:9]:
```suggestion
Step 1:  Click the "Open in Github Codespaces" button above and then click Fork!
```

Comment (blackary) [reply README.md:9]: Just kidding, github doesn't allow that :(

File: README.md (hunk 5 above)
Comment (blackary) [inline README.md]:
```suggestion
Step 5: Go back to the Codespaces instance, and add a new file in the pages/ folder called `my_chart.py`.
```

**Review by blackary — APPROVED** [review]
Thanks, Tyler!

**Pull request merged**